SmartDeviceLink/SDLAppInterfaceUnregisteredReason.h

```diff
@@ -73,3 +73,9 @@ extern SDLAppInterfaceUnregisteredReason const SDLAppInterfaceUnregisteredReason
 ///
 /// @since RPC 4.1
 extern SDLAppInterfaceUnregisteredReason const SDLAppInterfaceUnregisteredReasonUnsupportedHMIResource;
+
+
+/// By getting this value, SDL should unregister the application to allow the HMI to close the application.
+///
+/// @since RPC 6.0
```

Comment:
I think this description needs to be updated to match the requested description change in rpc_spec issue https://github.com/smartdevicelink/rpc_spec/pull/270. Also version number should be 7.0